gufe/components/smallmoleculecomponent.py

```diff
@@ -223,6 +237,7 @@ def _to_dict(self) -> dict:
                     _ATOMCHIRAL_TO_INT[atom.GetChiralTag()],
                     atom.GetAtomMapNum(),
                     atom.GetPropsAsDict(includePrivate=False),
+                    _HYBRIDIZATION_TO_INT[atom.GetHybridization()],
```

Comment:
What happens in this case if this isn't set?

I.e. if you read and old file (which now skips the hybridization reading), then write it out again - what does this pick up?